acme/setup.py

```diff
@@ -22,15 +22,14 @@
 ]
 
 # env markers in extras_require cause problems with older pip: #517
+# Keep in sync with conditional_requirements.py.
 if sys.version_info < (2, 7):
     install_requires.extend([
         # only some distros recognize stdlib argparse as already satisfying
         'argparse',
-        'mock<1.1.0',
```

Comment:
I don't really agree that we need to split tests into separate packages before removing `mock` as a dependency (I think it's not insane to have a package that functions with core functionality without some extra package X, but which can do more -- such as run tests -- if X is present).

But I do agree that there's some risk of introducing bugs this way.  So my vote would be pin `mock==1.1.0` for this release, then take more time to consider possible breakage cases, perhaps wrap `import mock`s in an error handler, and then revert the `mock==1.1.0` pin for a future release?